### PR TITLE
solution

### PR DESCRIPTION
**Pull request by MaxSchmide**

### CODE REVIEW

**Review by NataliGru — APPROVED** [review]
Great job!🔥🔥🔥

File: services/expenses.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+};
+
+const removeExpense = (expenseId) => {
+  const expenseIndex = expenses.findIndex(({ id }) => id === +expenseId);
+
+  expenses.splice(expenseIndex, 1);
+};
+
+const clearExpenses = () => {
+  expenses.length = 0;
```
Comment (anstsot) [inline services/expenses.js:55]: better to reassign empty array
```suggestion
  expenses = [];
```

File: services/users.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+};
+
+const removeUser = (userId) => {
+  const foundedIndex = users.findIndex(({ id }) => id === +userId);
+
+  users.splice(foundedIndex, 1);
+};
+
+const clearUsers = () => {
+  users.length = 0;
```
Comment (anstsot) [inline services/users.js:37]: same here
```suggestion
  users = [];
```

File: controllers/expenses.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+  res.status(200).send(expenses);
+};
+
+const getExpense = (req, res) => {
+  const { expenseId } = req.params;
+
+  if (isNaN(+expenseId)) {
+    res.status(400).send('Bad request');
+  }
```
Comment (anstsot) [inline controllers/expenses.js:24]: don't forget to stop the code
```suggestion
    res.status(400).send('Bad request');
    
    return;
  }
```

File: controllers/expenses.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  ];
+
+  const isFieldMissing = expenseFields.some(field => {
+    return !(field in body) || !body[field].toString().length;
+  });
+
+  const foundedUser = usersService.getUserById(body.userId);
+
+  if (isFieldMissing) {
+    res.status(400).send('Some field is missing');
```
Comment (anstsot) [inline controllers/expenses.js:54]:
```suggestion
    res.status(400).send('Some field is missing');
    return;
```

File: controllers/expenses.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  }
+
+  if (!foundedUser) {
+    res.status(400).send('User not found');
```
Comment (anstsot) [inline controllers/expenses.js:58]:
```suggestion
    res.status(400).send('User not found');
    return; 
```

**Pull request closed without merging**